ui/components/MesheryPerformanceComponent.js

```diff
@@ -111,17 +111,43 @@ class MesheryPerformanceComponent extends React.Component {
     };
   }
 
-  handleChange = (name) => (event) => {
-    if (name === 'url' && event.target.value !== '') {
-      this.setState({ urlError: false });
-    }
-    if (name === 't' && (event.target.value.toLowerCase().endsWith('h')
-      || event.target.value.toLowerCase().endsWith('m') || event.target.value.toLowerCase().endsWith('s'))) {
-      this.setState({ tError: false });
-    }
+  // handleChange = (name) => (event) => {
 
-    this.setState({ [name]: event.target.value });
-  };
+  //   if (name === 'url' && event.target.value !== '') {
+  //     this.setState({ urlError: false });
+  //   }
+  //   if (name === 't' && (event.target.value.toLowerCase().endsWith('h')
+  //     || event.target.value.toLowerCase().endsWith('m') || event.target.value.toLowerCase().endsWith('s'))) {
+  //     this.setState({ tError: false });
+  //   }
+
+  //   this.setState({ [name]: event.target.value });
+  // };
+
+  handleChange = (name) => (event) => {
+    const value = event.target.value;
+   if(name === 'url' ){
+       if (value){
+           if (value.substring(0,4) === "http" && value.substring(0,7) === "http://"){
```

Comment:
Just use string include property to check presence of http or https.

Reply:
We can do that too. But I think this should also work. If it is missing something then please let me know.

Reply:
It's nice to see this PR come through, @rshivam08 👍 

FYI - A future version of of this performance test form will likely segregate protocol from endpoint name in that protocol (http | https | grpc | nats) is likely to become a radio button next to the endpoint field.

Reply:
Development is all about optimisation, there can be a lot of approached for a single problem. Right now you have used comparison 4 times which is not a optimal approach, and is difficult to scale. Rather than using string.include, we should use a regex for valid url protocol.

Reply:
@kushthedude, I got your point. I will do it by using regex.

Reply:
That'd be great!